python/lsst/sims/catalogs/generation/db/ObservationMetaData.py

```diff
@@ -139,76 +163,311 @@ def __init__(self, boundType=None, boundLength=None,
                     raise RuntimeError('The list of m5s you passed to ObservationMetaData ' + \
                                        'has a different length than the list of bandpass names you passed')
 
             #now build the m5 dict
             if bandpassIsList:
-                self.m5 = {}
-                for b, m in zip(self.bandpass, m5):
-                    self.m5[b] = m
+                if len(m5) != len(self._bandpass):
+                    raise RuntimeError('In ObservationMetaData you tried to assign bandpass ' +
+                                       'and m5 with lists of different length')
+
+                self._m5 = {}
+                for b, m in zip(self._bandpass, m5):
+                    self._m5[b] = m
             else:
-                self.m5 = {self.bandpass:m5}
+                self._m5 = {self._bandpass:m5}
 
-    @property
-    def summary(self):
-        mydict = {}
-        mydict['site'] = self.site
 
-        mydict['boundType'] = self.boundType
-        mydict['boundLength'] = self.boundLength
-        mydict['unrefractedRA'] = self.unrefractedRA
-        mydict['unrefractedDec'] = self.unrefractedDec
-        mydict['rotSkyPos'] = self.rotSkyPos
+    def _buildBounds(self):
+        """
+        Set up the member variable self._bounds.
 
-        mydict['mjd'] = self.mjd
-        mydict['bandpass'] = self.bandpass
-        mydict['skyBrightness'] = self.skyBrightness
-        # mydict['m5'] = self.m5
+        If self._boundType, self._boundLength, self._unrefractedRA, or
+        self._unrefractedDec are None, nothing will happen.
+        """
 
-        mydict['phoSimMetadata'] = self.phoSimMetadata
+        if self._boundType is None:
+            return
 
-        return mydict
+        if self._boundLength is None:
+            return
 
-    def buildBounds(self):
-        if self.boundType is None:
+        if self._unrefractedRA is None or self._unrefractedDec is None:
             return
 
-        if self.boundLength is None:
-            raise RuntimeError("ObservationMetadata cannot assign a bounds; it has no boundLength")
+        self._bounds = SpatialBounds.getSpatialBounds(self._boundType, self._unrefractedRA, self._unrefractedDec,
+                                                     self._boundLength)
 
-        if self.unrefractedRA is None or self.unrefractedDec is None:
-            raise RuntimeError("ObservationMetadata cannot assign a bounds; it has no unrefractedRA/Dec")
+    def _assignPhoSimMetaData(self, metaData):
+        """
+        Assign the dict metaData to be the associated phoSimMetaData dict of this object.
 
-        self.bounds = SpatialBounds.getSpatialBounds(self.boundType, self.unrefractedRA, self.unrefractedDec,
-                                                     self.boundLength)
+        In doing so, this method will copy unrefractedRA, unrefractedDec, rotSkyPos,
+        MJD, and bandpass from the phoSimMetaData (if present) to the corresponding
```

Comment:
Yes: we should probably specify what the contents of a phoSimMetaData dict should be.

No: I don't know what they are.

It is on my "to do" list to learn the ins and outs of PhoSim a bit better.  I will make updating this docstring a part of that task.